geometry_msgs/Pose pose
std_msgs/ColorRGBA color
string color_name
